### DISCUSSION TITLE
PostgreSQL support

### DISCUSSION THREAD

**Discussion by bronger** [juanluisbaptiste/docker-otrs#111] [Q&A] [answered]
It is possible to set PostgreSQL as the DB backend for https://hub.docker.com/r/juanluisbaptiste/znuny?

**Comment by juanluisbaptiste** [comment, marked as answer]
Hi,

Currently it does not and I do not plan to, but PR's are always welcome.

PS: `juanluisbaptiste/otrs` is still the "official" supported name, `juanluisbaptiste/znuny` only gets an automated build of `master` branch and no version tags, so if you want to use versioned images you should still use the otrs image.
